Changes

```diff
@@ -14,6 +14,9 @@ LIST OF CHANGES
  - Extended the code for the samplesheet daemon to generate samplesheets for
    NovaSeqX instruments.
  - Removed remaining unused test data from the era of xml st::api::lims driver.
+ - update signature and iRODS documentation link in template for emails
```

Comment:
Would be good to keep the same style for Changes. So update -> Updated, remove -> Removed